### DISCUSSION TITLE
TOKEN

### DISCUSSION THREAD

**Discussion by Sangamesh30** [community/community#56778] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Where to find the token we have generated already

**Comment by airtower-luna** [comment]
If you mean a PAT the answer is simply "you don't". The page where you see a freshly generated PAT is very clear about that.

**Comment by waelbenmustapha** [comment, marked as answer]
you can't access an old generated token , you have to regenerate it , you can follow this steps  . 

In the upper-right corner of the page, click on your profile picture or avatar. A dropdown menu will appear.

From the dropdown menu, select "Settings". You will be taken to your account settings page.

In the left sidebar, click on "Developer settings". A submenu will appear.

From the submenu, select "Personal access tokens". You will be directed to the personal access tokens page.

On the personal access tokens page, click on the "Generate new token" button. It is usually located on the top-right side of the page.

You will be prompted to provide a Token description. Enter a descriptive name for the token to help you identify its purpose later on.

Under "Select scopes", choose the appropriate permissions or access levels for the token. Make sure to only select the necessary scopes required for your intended usage to maintain security.

Once you have chosen the scopes, click on the "Generate token" button at the bottom of the page.

GitHub will generate a new personal access token for you.

**Reply by Sangamesh30** [reply]
Thank you 👍

**Reply by zxcvb51226** [reply]
> you can't access an old generated token , you have to regenerate it , you can follow this steps  . 
> 
> 
> 
> In the upper-right corner of the page, click on your profile picture or avatar. A dropdown menu will appear.
> 
> 
> 
> From the dropdown menu, select "Settings". You will be taken to your account settings page.
> 
> 
> 
> In the left sidebar, click on "Developer settings". A submenu will appear.
> 
> 
> 
> From the submenu, select "Personal access tokens". You will be directed to the personal access tokens page.
> 
> 
> 
> On the personal access tokens page, click on the "Generate new token" button. It is usually located on the top-right side of the page.
> 
> 
> 
> You will be prompted to provide a Token description. Enter a descriptive name for the token to help you identify its purpose later on.
> 
> 
> 
> Under "Select scopes", choose the appropriate permissions or access levels for the token. Make sure to only select the necessary scopes required for your intended usage to maintain security.
> 
> 
> 
> Once you have chosen the scopes, click on the "Generate token" button at the bottom of the page.
> 
> 
> 
> GitHub will generate a new personal access token for you. 
> 
> 
> 
>

**Reply by Jones098** [reply]
Hello 👋
